src/App.tsx

```diff
@@ -1,61 +1,120 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+ import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo, TodoWithUser } from './types/Todo';
+import { User } from './types/User';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+
+function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+};
+
+const todosWithUsers: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
Can cocncat this in 1 function and do Function Expression
const todosWithUsers: TodoWithUser[] = todosFromServer.map(todo => {
  const user =
    usersFromServer.find(person => person.id === todo.userId) || null;

  return {
    ...todo,
    user,
  };
});